.github/actions/github_modified_files/action.yaml

```diff
@@ -0,0 +1,43 @@
+name: Get Files Diff
```

Comment:
The Action is necessary, because otherwise it cannot be used as a step of a workflow.
The workflow is an addition, if it will never be used we can remove, I left it with the purpose of evaluating in this PR if can be useful to call it individually.

Reply:
that's right, but it looks like the workflow does not use the action atm but re-implements the same code

Reply:
yes you are right, my fault